applications/callflow/src/cf_exe.erl

```diff
@@ -500,6 +513,10 @@ handle_event(JObj, #state{cf_module_pid=PidRef
                                ,{'cf_event_pids', Others}
                               ]}
             end;
+        {{<<"call_event">>, <<"CHANNEL_DESTROY">>}, CallId} ->
```

Comment:
I think this case clause is unnecessary, it would be handled by line 534.  As such could you please remove this?